src/extensions/core-extension-api.ts

```diff
@@ -7,10 +7,12 @@ export * from "./lens-renderer-extension"
 import * as EventBus from "./core-api/event-bus"
 import * as Store from "./core-api/stores"
 import * as Util from "./core-api/utils"
+import * as App from "./core-api/app"
```

Comment:
BTW this file is actually moved to `extensions/core-api/index.ts`
But it seems it's still exists in base branch (removed in follow up PR: https://github.com/lensapp/lens/pull/1140)

Reply:
fixed and removed also here